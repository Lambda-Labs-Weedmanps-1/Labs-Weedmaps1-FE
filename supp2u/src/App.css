.Navigation {
  background-color: #8debac;
  height: 40px;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  color: white;
}
.Link {
  padding: 10px;
}
