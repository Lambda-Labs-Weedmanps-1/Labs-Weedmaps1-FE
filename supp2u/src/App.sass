.Navigation
  background-color: #8debac
  height: 40px
  display: flex
  flex-direction: row
  align-items: center
  justify-content: center
  font-size: calc(10px + 2vmin)
  color: white

.App
  margin: 0 auto
  display: flex
  flex-direction: column
  text-align: center
  overflow-x: hidden
  width: 1000px

.Map_Holder
  justify-content: center
  padding: 10px

.App-logo
  animation: App-logo-spin infinite 20s linear
  height: 40vmin
  pointer-events: none

.App-header
  background-color: #282c34
  min-height: 100vh

.Link
  padding: 10px