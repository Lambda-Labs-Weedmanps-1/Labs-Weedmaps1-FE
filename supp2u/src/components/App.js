import React, { Component, useState } from 'react';
// import logo from '../../assets/img/logo.svg';
import '../css/styles.styl';
import Home from './Home';
import { auth } from '../utils/init';

// * Hooks Implementation
const App = () => {
  // maybe status, isLoggedIn ???
  // const [isLoggedIn, logInUser] = useState(auth.loggedIn())
  // const [isLoggedIn, logInUser] = useState(auth.loggedIn())

  return (
    <div className="App">
      <div className="App-header">
        {/* <img src={logo} className="App-logo" alt="logo" /> */}
        <h2>Welcome to React + Auth0 + Rails API</h2>
      </div>
      <Home
        auth={auth}
        // isLoggedIn={logInUser}
        // isLoggedIn={logInUser(sisLoggedIn)}
        token={auth.getToken()}
      />
    </div>
  );
}


// class App extends Component {
// 	constructor(props) {
// 		super(props);
// 		this.state = {
// 			isLoggedIn: auth.loggedIn()
// 		};
// 	}
// 	render() {
// 		return (
// 			<div className="App">
// 				<div className="App-header">
// 					{/* <img src={logo} className="App-logo" alt="logo" /> */}
// 					<h2>Welcome to React + Auth0 + Rails API</h2>
// 				</div>
// 				<Home
// 					auth={auth}
// 					isLoggedIn={this.state.isLoggedIn}
// 					token={auth.getToken()}
// 				/>
// 			</div>
// 		);
// 	}
// }
export default App;
