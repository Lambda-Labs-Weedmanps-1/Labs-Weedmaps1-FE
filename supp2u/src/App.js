import React from 'react';
import Navigation from './components/Navigation/Navigation'
import './App.sass';
import BasicRoute from "./BasicRoute";

import { toast } from 'react-toastify';
import 'react-toastify/dist/ReactToastify.min.css';
toast.configure();


function App() {
  return (
    <div className="App">
        <Navigation />
        <BasicRoute />
    </div>
  );
}

export default App;
