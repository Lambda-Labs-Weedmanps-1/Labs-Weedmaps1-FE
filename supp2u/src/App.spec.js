import React from 'react';
import App from './App';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
// this is an extremely simple test to just see if the app is rendering at all, if this fails all is lost
describe('APP', ()=>{
    it ('Renders without crashing', ()=>{
        const application = shallow(<App />);
    
      expect(application).toMatchSnapshot();
    });
})