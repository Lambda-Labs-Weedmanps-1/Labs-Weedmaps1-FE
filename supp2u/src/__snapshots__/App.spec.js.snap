// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`APP Renders without crashing 1`] = `
<div
  className="App"
>
  <withRouter(Navigation) />
  <_default />
</div>
`;
